lib/chef/knife/bmcs_server_list.rb

```diff
@@ -25,10 +27,31 @@ def run
         options = {}
         options[:limit] = config[:limit] if config[:limit]
 
-        response = compute_client.list_instances(compartment_id, options)
+        servers = {}
 
-        display_list(response, ['Display Name', 'ID', 'State']) do |item|
-          [item.display_name, item.id, item.lifecycle_state]
+        instances = compute_client.list_instances(compartment_id, options)
+
+        # build a hash of hashs, keyed off of server_id
+        vnics = compute_client.list_vnic_attachments(compartment_id, {})
```

Comment:
We want to be able to run the unit tests without hitting an actual service, so you'll need to mock these new calls in the tests. (Check by clearing any KNIFE_BMCS_* environment variables and "rake" should still pass.)